cyral/tests/test_cyral.py

```diff
@@ -0,0 +1,52 @@
+import mock
+
+from datadog_checks.cyral.cyral import CyralCheck
+
+CHECK_NAME = 'cyral'
+
+
+data = """# HELP cyral_analysis_time Time spent in just doing analysis
```

Comment:
Let's move this data output into a text file under a `cyral/tests/fixtures/metrics.txt` and refactor the mock requests to a fixture:

https://github.com/DataDog/integrations-core/blob/8c6b38880732eca9eeb07353fb421e22cd4b84e7/cilium/tests/conftest.py#L79-L90

Reply:
Thanks for the tip!

I just took a whack at this in https://github.com/DataDog/integrations-extras/pull/658/commits/0a27b2205497e66e863e33f331ab4b5c3ec6de61, but caveat, 😄 I'm not a Python programmer, I'm actually a Go programmer. Please let me know if I need to change it more, and feel free to give me the "For Dummies" version of instructions because Python isn't my daily language.

Reply:
No problem! I can share some references that may help:

Spec.yaml/conf.yaml.example docs: https://datadoghq.dev/integrations-core/meta/config-specs/

OpenMetricsBaseCheck docs: https://datadoghq.dev/integrations-core/base/prometheus/

Let me know if you have any questions!